### ISSUE TITLE
Real hasSymbolData exactly 1 xsd:string

### ISSUE THREAD

**Issue by urbanmatthias** [emmo-repo/EMMO#85]
In the 1.0.0-alpha2 version of emmo a `math.Real hasSymbolData exactly 1 xsd:string`, because it is a subclass of symbol. I think it should only have numerical data. What do you think?

![image](https://user-images.githubusercontent.com/42069939/92703763-c0754480-f352-11ea-9b70-062c0cac362e.png)

**Comment by emanueleghedini** [comment]
You're right.

It comes from the fact that a number in the EMMO is a Symbol for the Mathematical language (i.e. is an elementary mark for mathematical expressions), and inherits the Symbol axiom of having a xsd:string value.

In the real world a real number is a sub-type of string (i.e. is a string that you interpret as a number), but for OWL they are two different classes: xsd:string and xsd:double. This confusion between common sense and actual xsd datatype hierarchy led to the definition of the above mentioned axiom.

Assuming that each Symbol must have a xsd:string value seems wrong, and the axiom will be removed.

Thanks.

**Comment by jesper-friis** [comment]
I am closing this issue since it has been merged into 1.0.0-alpha2

**Comment by urbanmatthias** [comment]
Hi, I have a question concerning this issue.

`hasSymbolData` has domain Symbol and is functional. That still means that every Symbol, including integers or doubles must have that relationship, right?

```
    <owl:DatatypeProperty rdf:about="http://emmo.info/emmo/middle/perceptual#EMMO_23b579e1_8088_45b5_9975_064014026c42">
        <rdf:type rdf:resource="http://www.w3.org/2002/07/owl#FunctionalProperty"/>  # it is functional
        <rdfs:domain rdf:resource="http://emmo.info/emmo/middle/perceptual#EMMO_a1083d0a_c1fb_471f_8e20_a98f881ad527"/>  # domain is symbol
        <rdfs:range rdf:resource="http://www.w3.org/2001/XMLSchema#string"/>
        <skos:prefLabel xml:lang="en">hasSymbolData</skos:prefLabel>
    </owl:DatatypeProperty>
```
![Image Pasted at 2020-9-23 16-39](https://user-images.githubusercontent.com/42069939/94028146-e6233480-fdbb-11ea-8da2-b129e3bca938.png)
